application/controllers/template_controller.py

```diff
@@ -62,3 +74,14 @@ def get_comms_template_by_id(template_id):
     if not template:
         logger.info("Tried to GET non-existent template with id", id=template_id)
     return template.to_dict() if template else template
+
+
+def get_comms_templates_by_classifiers(classifiers):
+    templates = get_templates_by_classifiers(classifiers)
+    if templates:
```

Comment:
My personal preference is to use try/except blocks over if/else where possible. I.e IEAFTP than LBYL.